<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bulb</class>
 <widget class="QMainWindow" name="Bulb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voltage Input Validator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblVoltageRange">
      <property name="text">
       <string>Enter voltage between 1 and 240V:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="voltageLineEdit">
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbVoltageTest">
      <property name="text">
       <string>Voltage test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
